### ISSUE TITLE
Unclear issue with konva in firefox.

### ISSUE THREAD

**Issue by dooddy** [konvajs/konva#98]
http://codepen.io/anon/pen/XmpogK
Here is example which good works in chrome, but partially works in firefox. Do you have any idea about it?

**Comment by lavrton** [comment]
Hi. Nice demo. I tried to debug in firefox. Damn... Firefox's debugger looks so bad after Chrome. Not luck yet.

**Comment by dooddy** [comment]
Thanks. I hope you'll find the cause

**Comment by lavrton** [comment]
Hi. I was trying to take a look into this issue again. But I have no idea why it is not working.

I have firefox 41.0.1 and Ubuntu system. I tried to draw directly into canvas with simple native canvas functions like `fillRect` but no changes are visible. Looks like it is bug with firefox. As I tested your demo via https://www.browserling.com/ with firefox 41 and windows system, and, you what, it is working...
